NitroxPatcher/Patches/Dynamic/PingManager_NotifyRename_Patch.cs

```diff
@@ -16,11 +19,20 @@ public class PingManager_NotifyRename_Patch : NitroxPatch, IDynamicPatch
 
         public static void Postfix(PingInstance instance)
         {
-            if (!instance)
+            // Only beacons are synced here (not mission, vehicle or other signals) because spawning is handled differently for non-droppable entities
+            if (!instance || !instance.GetComponent<Beacon>())
             {
                 return;
             }
-            NitroxServiceLocator.LocateService<IPacketSender>().Send(new PingRenamed(NitroxEntity.GetId(instance.gameObject), instance.GetLabel(), SerializationHelper.GetBytes(instance.gameObject)));
+            NitroxId nitroxId = NitroxEntity.GetId(instance.gameObject);
+            if (nitroxId == null)
+            {
+                Log.Error($"Missing Nitrox id on beacon: {instance.GetHierarchyPath()}");
+                return;
+            }
```

Comment:
I don't think this is possible.  The function above will always return an id - if one is not set, it will set one,

Reply:
Might be good to keep it in debug mode then?

Reply:
I think the assumption that this always returns a value is throughout the code base.

Reply:
Yeah just thought the same. If it's not working it will fail everywhere. Fixed.